docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -400,22 +405,25 @@ quarkus.oidc.authentication.error-path=/error
 ----
 
 Ensure that the property starts with a forward slash (/) character and the path is relative to the base URI of the current endpoint.
-For example, if it is set as '/error' and the current request URI is `https://localhost:8080/callback?error=invalid_scope` then a final redirect will be made to `https://localhost:8080/error?error=invalid_scope`.
+For example, if it is set to '/error' and the current request URI is `https://localhost:8080/callback?error=invalid_scope`, then a final redirect is made to `https://localhost:8080/error?error=invalid_scope`.
 
-It is important to ensure that this error endpoint is a public resource to prevent the user from being redirected to this page to be authenticated again.
+[IMPORTANT]
+====
+To prevent the user from being redirected to this page to be reauthenticated, ensure that this error endpoint is a public resource.
```

Comment:
```suggestion
Ensure that this error endpoint is a public resource to prevent the user from being redirected to this page to be re-authenticated.
```

Reply:
@MichalMaler, the sequence according to IBM style should be "what" before "how". Therefore, I propose to keep this as is, that is, "To prevent......, ensure that....". Good catch though on the missing hyphen in "re-authenticated". Fixed this.